api/application/client.go

```diff
@@ -162,7 +163,20 @@ func (c *Client) Deploy(args DeployArgs) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	return errors.Trace(results.OneError())
+	err = results.OneError()
+	if err == nil {
+		return nil
+	}
+	if pErr, ok := errors.Cause(err).(*params.Error); ok {
+		switch pErr.Code {
+		case params.CodeAlreadyExists:
+			// Remove the "already exists" in the error message to prevent
+			// stuttering.
+			msg := strings.Replace(err.Error(), " already exists", "", -1)
+			return errors.AlreadyExistsf(msg)
+		}
+	}
+	return errors.Trace(err)
```

Comment:
This code is not in develop

Reply:
I added this, so we can better present an error message.